src/index.html

```diff
@@ -5,10 +5,112 @@
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
-    <title>Moyo header</title>
+    <title>moyo.ua</title>
+    <link rel="icon" href="./images/logo.png">
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+
+    <header class="menu-container">
+
+      <a href="./index.html"> 
+        <img
+          src="./images/logo.png"
+          alt="logo images"
+          class="logo"
+        >
+      </a>
+
+      <nav class="nav-container">
+
+        <ul class="nav-list">
+          
+          <li class="nav-marker">
+
+            <a
```

Comment:
```suggestion
      <nav class="nav-container">
        <ul class="nav-list">
          <li class="nav-marker">
            <a
```
 blank line shouldn't be between parent and child elements. just between siblings
 please check all cases